content/en/_index.md

```diff
@@ -49,32 +48,37 @@ OpenTelemetry is a collection of APIs, SDKs, and tools. Use it to instrument,
 generate, collect, and export telemetry data (metrics, logs, and traces) to help
 you analyze your software's performance and behavior.
 
-> OpenTelemetry is **generally available** across
-> [several languages](docs/languages/) and is suitable for use.
+> OpenTelemetry is [generally available](/status/) across
+> [several languages](docs/languages/) and is suitable for production use.
 
 {{% /blocks/lead %}}
 
 {{% blocks/section color="dark" type="row" %}}
 
-{{% blocks/feature icon="fas fa-chart-line" title="Traces, Metrics, Logs"%}}
+{{% blocks/feature icon="fas fa-chart-line" title="Traces, Metrics, Logs" url="docs/concepts/observability-primer/" %}}
 
 Create and collect telemetry data from your services and software, then forward
-them to a variety of analysis tools. {{% /blocks/feature %}}
+them to a variety of analysis tools.
 
-{{% blocks/feature icon="fas fa-magic" title="Drop-In Instrumentation"%}}
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fas fa-magic" title="Drop-in Instrumentation & Integrations" %}}
```

Comment:
If we remove the "Drop-in", can we get the heading all on one line? If so, I think it's worth removing to balance the three blocks.

Reply:
In a sense, what I understood from the original intent (cc @austinlparker @tedsuo) was that the use of "drop-in" was purposeful here, and part of the "marketing" tag line :).

Any thoughts from others about this? @theletterf @svrnm @cartermp